modules/task/execution.go

```diff
@@ -133,6 +133,15 @@ func Register(fn interface{}) error {
 	return nil
 }
 
+// MustRegister registers a function for an async task or panics
+// Since Task registration is performed in main() we want to fail-fast and
+// reduce the amount of error-checking
+func MustRegister(fn interface{}) {
```

Comment:
Interesting, I'll add one. Yet another reason why I think this is valuable -- fail fast in main if stuff isn't set up right rather than logging and doing... what exactly?